sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -0,0 +1,127 @@
+# Preliminary Keystone Instructions
```

Comment:
Just a note -- I was starting to write one myself, but see that you've done it yourself. So, thanks!

(I didn't review the instructions, assuming that you two have covered it for accuracy.)